test/integration/krane_test.rb

```diff
@@ -3,7 +3,9 @@
 
 class KraneTest < Krane::IntegrationTest
   def test_restart_black_box
-    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "web.yml.erb", "redis.yml"]))
+    assert_deploy_success(
+      deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "web.yml.erb", "redis.yml"], render_erb: true)
```

Comment:
@dturn is there an open issue to refactor tests to reflect the new 'deploy can't render' world? It feels wrong to be able to set `render_erb` here (though I understand now is probably not the time to fix this)

Reply:
No but I had planned on opening one.